packages/dds/tree/src/core/tree/anchorSet.ts

```diff
@@ -401,56 +399,26 @@ export class AnchorSet implements ISubscribable<AnchorSetRootEvents> {
 	}
 
 	/**
-	 * Updates paths for a range move (including re-parenting path items and updating indexes).
-	 * @param count - number of siblings to insert/delete/move.
-	 * @param srcStart - where the siblings are removed from. If undefined the operation is an insert.
-	 * @param dst - where the siblings are moved to. If undefined the operation is a delete.
-	 *
-	 * TODO:
-	 * How should anchors that become invalid, then valid again (ex: into content that was deleted, then undone) work?
-	 * Add an API to resurrect them? Store them in special detached fields? Store them in special non-detached fields?
-	 *
-	 * TODO:
-	 * How should custom anchors work (ex: ones not just tied to a specific Node)?
-	 * This design assumes they can be expressed in terms of a Node anchor + some extra stuff,
-	 * but we don't have an API for the extra stuff yet.
-	 *
-	 * TODO: tests
```

Comment:
If you are going to delete the TODO that this function should have tests and replace it with multiple new functions, they should probably have unit tests (or at the very least have todods that they should have unit tests)